batchglm/train/numpy/glm_norm/estimator.py

```diff
@@ -0,0 +1,77 @@
+import logging
+
+import numpy as np
+
+from .external import EstimatorGlm, Model
+from .model_container import ModelContainer
+from ....models.glm_norm.utils import init_par
+logger = logging.getLogger("batchglm")
+
+
+class Estimator(EstimatorGlm):
+
+
+    def __init__(
+        self,
+        model: Model,
+        init_location: str = "AUTO",
+        init_scale: str = "AUTO",
+        # batch_size: Optional[Union[Tuple[int, int], int]] = None,
+        quick_scale: bool = False,
+        dtype: str = "float64",
+    ):
+        """
+        Performs initialisation and creates a new estimator.
+        :param model:
+            The GLM model to be fit
+        :param init_location: (Optional)
+            Low-level initial values for a. Can be:
+
+            - str:
+                * "auto": automatically choose best initialization
+                * "standard": initialize intercept with observed mean
+                * "closed_form": try to initialize with closed form
+            - np.ndarray: direct initialization of 'a'
+        :param init_scale: (Optional)
+            Low-level initial values for b. Can be:
+
+            - str:
+                * "auto": automatically choose best initialization
+                * "random": initialize with random values
+                * "standard": initialize with zeros
+                * "closed_form": try to initialize with closed form
+            - np.ndarray: direct initialization of 'b'
+        :param quick_scale: bool
+            Whether `scale` will be fitted faster and maybe less accurate.
+            Useful in scenarios where fitting the exact `scale` is not absolutely necessary.
+        :param dtype: Numerical precision.
```

Comment:
please check the documented values for `init_location` and `init_scale`. It seems not all of them are implemented in `init_par`.

Reply:
Yes I added a warning about closed form for location.  If we fit the model there, then `train` doesn't actually have to do anything.

Reply:
ok, mark as resolved then I guess?